locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,72 @@
+# Strings for the breach details checklists
```

Comment:
I've been breaking out the .ftl files to match the naming conventions used across the repo.  So in addition to `app.ftl` I recently added `landing.ftl` which includes all landing page copy.  Should we continue this pattern here?  This file would be named `breaches.ftl` corresponding with the breaches page, and any other related breaches copy (there shouldn't be much more) would also be added to this page...

Reply:
Done, thanks